ansible/roles/app/templates/support.conf.j2

```diff
@@ -8,51 +8,25 @@ Listen {{ apache_app_listen_address.support }}:{{ loadbalancing.support.port }}
     ServerAdmin {{ apache_server_admin }}
     Options -indexes +Includes
     DocumentRoot {{ document_root }}
-    Alias /registration {{ support_data_dir }}/current/web
-    SetEnv HTTPS on
 
-    <Directory "{{ support_data_dir }}">
+    <Directory {{ document_root }}>
```

Comment:
I'd like to see that the variable {{ document_root }} is changed to something less generic and more SP Dashboard specific to make it usable in the OpenConext deploy playbooks.